pyoptsparse/pyIPOPT/pyIPOPT.py

```diff
@@ -281,8 +281,8 @@ def intermediate(self_cyipopt, alg_mod, iter_count, obj_value, inf_pr, inf_du, m
 
                         # Find pyoptsparse call counters for objective and constraints calls at current x.
                         # IPOPT calls objective and constraints separately, so we find two call counters and append iter_dict to both counters.
-                        call_counter_1 = self.hist._searchCallCounter(self.cache["x"])
-                        call_counter_2 = self.hist._searchCallCounter(self.cache["x"], last=call_counter_1 - 1)
+                        call_counter_1 = self.hist._searchCallCounter(self.optProb._mapXtoUser(self.cache["x"]))
```

Comment:
Actually, I wonder if we should have some checks in case even with scaling we fail to find the call counter. In such cases we should be able to continue gracefully without raising errors

Reply:
Modified here to keep going if the matching call counter cannot be found and print a warning.